src/points/ActivityCard.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { fireEvent, render } from '@testing-library/react-native'
+import React from 'react'
+import { PointsEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import SwapArrows from 'src/icons/SwapArrows'
+import ActivityCard, { Props } from './ActivityCard'
+
+jest.mock('src/analytics/ValoraAnalytics')
+
+describe('ActivityCard', () => {
```

Comment:
i think you can add a `jest.clearAllMocks()` in a `beforeEach` since you check whether mocks have been called in the tests below (without this, the assertions could be on calls to that mock from previous tests)

Reply:
great catch, thanks!